### PR TITLE
Railsy forms

### PR DESCRIPTION
**Pull request by dev1980**

### CODE REVIEW

File: app/controllers/users_controller.rb [hunk 1]
```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+
+class UsersController < ApplicationController
+  def new
+    @user = User.new
+  end
+
+  def create
+    # @user = User.new(username: params[:username], email: params[:email], password: params[:password])
```
Comment (azdrenymeri) [inline app/controllers/users_controller.rb]: remove this comment

File: app/controllers/users_controller.rb [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+    end
+  end
+
+  def edit
+    @user = User.find(params[:id])
+  end
+
+  def update
+    @user = User.find(params[:id])
+    if @user.save
```
Comment (azdrenymeri) [inline app/controllers/users_controller.rb]: you can simply check `if @user.update(user_params)` you don't need to call `@user.save`

File: app/views/users/new.html.erb [hunk 3]
```diff
@@ -0,0 +1,45 @@
+<!-- 
+<form action="/users" method="post" name="users" accept-charset="UTF-8">
```
Comment (azdrenymeri) [inline app/views/users/new.html.erb]: you should use `users_path` for attribute `action`

File: app/views/users/new.html.erb [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+    <input id="email" name="users[email]" type="email"/><br>
+
+    <label for="users[password]">Password:</label><br>
+    <input id="password" name="users[password]" type="password"/><br>
+
+    <input id="submit" type="submit" value="Create User"/>
+</form>
+-->
+<!-- 
+ <%= form_tag("/users", method: "post") do %>
```
Comment (azdrenymeri) [inline app/views/users/new.html.erb]: same as with the first form you should use the rails helpers `users_path` as the first parameter of the `form_tag()` helper

File: app/controllers/users_controller.rb (hunk 1 above)
Comment (mapra99) [reply app/controllers/users_controller.rb]: This @user call corresponds to the second form (form_tag). We commented it following the instructions from OdinProject

File: app/controllers/users_controller.rb [hunk 5]
```diff
@@ -22,8 +23,7 @@ def edit
 
   def update
     @user = User.find(params[:id])
-    if @user.save
-      @user.update(user_params)
+    if @user.update(user_params)
```
Comment (samgaco) [inline app/controllers/users_controller.rb]: :+1:

**Pull request merged**